Integrations/integration-Cisco-umbrella.yml

```diff
@@ -1,11 +1,11 @@
 commonfields:
-  id: Cisco Umbrella Investigate
+  id: Cisco Umbrella Investigate - Python
```

Comment:
Why Python?

Reply:
To differentiate from the old integration (JS) that might still be active for clients.

Reply:
Why don't we replace the old one?

Reply:
We should make sure to be fully backward compatible, and then we should replace the old integration all together

Reply:
Honestly the only reason NOT to, is due to the old integration outputting several keys to context that are simply wrong, if someone uses/used it AS IS, it might not work for him.
Example: Domain.FirstSeean I fixed to Domain.FirstSeen (I couldn't not keep it with a mistake for the new integration)

Reply:
It is fully backward compatible, I will simply "duplicate" the old keys that are wrong to be supported but not declared in the settings output.